vaultlocker/vault_exceptions.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please could this module be named 'exceptions' - its already scoped by being part of the vaultlocker module